duckduckgo/iOS
# Updates to Fireproofing dialogue and Settings

Task/Issue URL: https://app.asana.com/0/72649045549333/1201479811018830/f
Tech Design URL:
CC: @miasma13 

**Description**:
This PR updates copy for Fireproof dialogue and Settings

**Steps to test this PR**:
1. From the home screen, login to a website that will trigger the Fireproof prompt suggestion alert dialogue e.g. amazon.com. Confirm dialogue now contains a title and message, and updated FIreproof button title as per design
2. Via the overflow menu, select the 'Fireproof This Site' option and confirm it shows the same updated alert dialogue as in Step 1
3. In Settings -> Fireproof Sites, confirm section header text above the toggle has been removed and that the toggle copy is correct. Also confirm that the section header text for the list of websites is correct
4. Remove all fireproof websites (if any) and confirm that when there are no Fireproof sites saved that placeholder copy is correct



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
